src/client/bedrock.rs

```diff
@@ -26,8 +26,30 @@ pub struct BedrockConfig {
 }
 
 impl BedrockClient {
-    config_get_fn!(access_key_id, get_access_key_id);
-    config_get_fn!(secret_access_key, get_secret_access_key);
+    // Removed macro to allow environment variable priority for access keys
+    // config_get_fn!(access_key_id, get_access_key_id);
+    // config_get_fn!(secret_access_key, get_secret_access_key);
+
+    pub fn get_access_key_id(&self) -> Result<String> {
+        if let Ok(val) = std::env::var("AWS_ACCESS_KEY_ID") {
+            return Ok(val);
+        }
+        if let Some(val) = &self.config.access_key_id {
+            return Ok(val.clone());
+        }
+        bail!("AWS_ACCESS_KEY_ID not set in environment or config");
+    }
+
+    pub fn get_secret_access_key(&self) -> Result<String> {
+        if let Ok(val) = std::env::var("AWS_SECRET_ACCESS_KEY") {
+            return Ok(val);
+        }
+        if let Some(val) = &self.config.secret_access_key {
+            return Ok(val.clone());
+        }
+        bail!("AWS_SECRET_ACCESS_KEY not set in environment or config");
+    }
+
     config_get_fn!(region, get_region);
```

Comment:
Please do not make any changes here. #1361 ensures the priority of environment variables.